third_party/istio-latest/install-istio.sh

```diff
@@ -18,6 +18,7 @@
 ISTIO_VERSION=1.7.3
 ISTIO_TARBALL=istio-${ISTIO_VERSION}-linux-amd64.tar.gz
 DOWNLOAD_URL=https://github.com/istio/istio/releases/download/${ISTIO_VERSION}/${ISTIO_TARBALL}
+SYSTEM_NAMESPACE="${SYSTEM_NAMESPACE:-"knative-serving"}"
```

Comment:
why do we need to have a different system namespace? is it because we want to have a new namespace without cluster local gateway installed?

Reply:
Thank you. (I made a mistake in `kubectl patch` below and fixed.)

This is because serving's e2e test deploys `config-istio` in the `SYSTEM_NAMESPACE` namespace 
https://github.com/knative/serving/blob/cbcd2ba4839711eafe1f846b6d886c84e8f56058/test/e2e-networking-library.sh#L62-L63 so we need to modify the `config-istio` in the namespace.